mobilitypath_visualizer/config/params.yaml

```diff
@@ -26,4 +26,6 @@ y: 0.5
 z: 1.0
 
 # Collision message display duration before disapearing(seconds)
-t: 3.0
\ No newline at end of file
+t: 3.0
+
+host_id:""
```

Comment:
This `host_id` parameter is actually loaded in from the `vehicle_id` parameter included in the /opt/carma/vehicle/calibration/identifiers/UniqueVehicleParams.yaml file. Since this file is passed at launch time from guidance.launch.py, we can remove the parameter from here.